Makefile.toml

```diff
@@ -14,6 +14,14 @@ BUILD_FLAVOR = "--release"
 BUILD_FLAVOR = "--release"
 TARGET = "--target=x86_64-unknown-linux-musl"
 
+[env.'release+mac']
+BUILD_FLAVOR = "--release"
+TARGET = "--target=x86_64-apple-darwin"
```

Comment:
should use `x86_64` instead of `aarch64`?

Reply:
Both, if possible, so we can support both x86 and M1 macs